site/pages/create-disruption.page.tsx

```diff
@@ -97,86 +213,100 @@ const CreateDisruption = (initialState: PageState<Partial<DisruptionPageInputs>>
+                        <Table rows={pageState.inputs.validity ? getValidityRows() : []} />
+                        {(pageState.inputs.validity || []).map((item, index) => (
+                            <Fragment key={`validity-${index}`}>
+                                <input type="hidden" name={`validity${index + 1}`} value={item.disruptionStartDate} />
+                                <input type="hidden" name={`validity${index + 1}`} value={item.disruptionStartTime} />
+                                <input type="hidden" name={`validity${index + 1}`} value={item.disruptionEndDate} />
+                                <input type="hidden" name={`validity${index + 1}`} value={item.disruptionEndTime} />
+                                <input
+                                    type="hidden"
+                                    name={`validity${index + 1}`}
+                                    value={item.disruptionNoEndDateTime}
+                                />
+                            </Fragment>
+                        ))}
 
-                        <DateSelector<DisruptionPageInputs>
+                        <DateSelector<Validity>
                             display="Start date"
                             hiddenHint="Enter in format DD/MM/YYYY"
-                            value={pageState.inputs.disruptionStartDate}
+                            value={validity.disruptionStartDate}
                             disabled={false}
                             disablePast={false}
-                            inputName="disruptionStartDate"
-                            stateUpdater={stateUpdater}
+                            inputName={"disruptionStartDate"}
+                            stateUpdater={validityStateUpdater}
                             initialErrors={pageState.errors}
-                            schema={createDisruptionSchema.shape.disruptionStartDate}
+                            reset={!validity.disruptionStartDate}
+                            schema={validitySchema.shape.disruptionStartDate}
                         />
 
-                        <TimeSelector<DisruptionPageInputs>
+                        <TimeSelector<Validity>
                             display="Start time"
                             hint="Enter the time in 24hr format. For example 0900 is 9am, 1730 is 5:30pm"
-                            value={pageState.inputs.disruptionStartTime}
+                            value={validity.disruptionStartTime}
                             disabled={false}
                             inputName="disruptionStartTime"
-                            stateUpdater={stateUpdater}
+                            stateUpdater={validityStateUpdater}
                             initialErrors={pageState.errors}
-                            schema={createDisruptionSchema.shape.disruptionStartTime}
+                            showError={validityButtonClicked && !validity.disruptionStartTime}
```

Comment:
This can be removed after removing `validityButtonClicked`

Reply:
done